econtext/src/test/java/com/adyen/checkout/econtext/internal/ui/EContextComponentTest.kt

```diff
@@ -6,7 +6,7 @@
  * Created by ozgur on 13/2/2023.
  */
 
-package com.adyen.checkout.econtext
+package com.adyen.checkout.econtext.internal.ui
```

Comment:
I don't think we do need to restructure packaging for component test class

Reply:
Whoops I think I switched up the component and delegate tests